lib/resources/json.rb

```diff
@@ -21,19 +30,28 @@ class JsonConfig < Inspec.resource(1)
 
     def initialize(path)
```

Comment:
should we rename `path` to `opts` then? Its only a path if used as a string then